templates/exports.j2

```diff
@@ -1,13 +1,3 @@
-# /etc/exports: the access control list for filesystems which may be exported
-#   to NFS clients.  See exports(5).
-#
-# Example for NFSv2 and NFSv3:
-# /srv/homes       hostname1(rw,sync,no_subtree_check) hostname2(ro,sync,no_subtree_check)
-#
-# Example for NFSv4:
-# /srv/nfs4        gss/krb5i(rw,sync,fsid=0,crossmnt,no_subtree_check)
-# /srv/nfs4/homes  gss/krb5i(rw,sync,no_subtree_check)
-#
 {% for export in nfs_exports %}
 {{ export }}
 {% endfor %}
```

Comment:
with those lines removed it's certain that code won't notify handler and nfs-server won't be started.